AUTHORS

```diff
@@ -45,3 +45,4 @@ Sarbagya Dhaubanjar <mail@sarbagyastha.com.np>
 Rody Davis Jr <rody.davis.jr@gmail.com>
 Robin Jespersen <info@unitedpartners.de>
 Jefferson Quesado <jeff.quesado@gmail.com>
+Mark Diener <rpzrpzrpz@gmail.com>
```

Comment:
Was this change from Mark?

Reply:
Yes, adapted from https://github.com/flutter/flutter/pull/40373 @markdiener

Reply:
I knew this looked familiar.